src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -69,6 +71,9 @@ const FieldRow = ({
             {label}
           </div>
           <div>{children}</div>
+          {(validationState && errorMessage) &&
+            <HelpBlock>{errorMessage}</HelpBlock>
+          }
```

Comment:
Why do we need this HelpBlock code duplication here and in lines 93-95?

Reply:
It's 2 variations of the `FieldRow` Component, so I thought that if we adding new feature to the horizontal (!vertical) variation we should add it to the vertical variation too.
Vertical version:
![image](https://user-images.githubusercontent.com/64131213/103342434-287ea080-4a92-11eb-93f6-e1368169f8fb.png)

I can extract it to function if you think it's better solution, pseudo code:
```
errorMessage  = () => (
    (validationState && errorMessage) &&
            <HelpBlock>{errorMessage}</HelpBlock>
)

```

```
      <React.Fragment>
        <Col offset={labelCols} cols={fieldCols} className={style['col-data']} id={id}>
          <div className={style['col-label-vertical']}>
            {label}
          </div>
          <div>{children}</div>
          {
               errorMessage()
          }
        </Col>
      </React.Fragment>
```